spec/components/create_a_job_sidebar_component_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe CreateAJobSidebarComponent, type: :component do
+  let(:vacancy) { create(:vacancy, completed_step: completed_step) }
+  let(:completed_step) { 0 }
+  let(:current_step) { 1 }
+  let(:school_group_user?) { true }
+
+  before do
+    allow_any_instance_of(HiringStaff::JobCreationHelper).to receive(:current_step).and_return(current_step)
+    allow_any_instance_of(HiringStaff::JobCreationHelper).to receive(:school_group_user?).and_return(school_group_user?)
+  end
+
+  it 'renders the sidebar' do
+    render_inline(described_class.new(vacancy: vacancy))
+
+    expect(rendered_component).to include('Create a job listing steps')
+  end
+
+  it 'renders active class on current_step' do
+    result = render_inline(described_class.new(vacancy: vacancy))
+
+    assert_includes(
```

Comment:
Is there a reason you sometimes use Rspec's `expect(fridge).to include(milk)` and sometimes Minitest's `assert_includes(fridge, milk)`?

Reply:
Just that I was following the docs